locations/spiders/stc_se.py

```diff
@@ -0,0 +1,18 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+class StcSESpider(scrapy.Spider):
+    name = "stc_se"
+    item_attributes = {"brand": "STC", "brand_wikidata": "Q124061743"}
+    start_urls = ["https://www.stc.se/assets/clubs.json"]
+
+    def parse(self, response):
+        for club in response.json():
+            item = DictParser.parse(club)
+            item["branch"] = item.pop("name")
+            item["city"] = item["city"].title()
```

Comment:
Looks like street should be `street_address`. This is one of the things where we deviate from OSM, it's not practical for us split it into `addr:street` and `addr:housenumber`/unit/housename/... in every spider. There are libraries that could help us, but that's a TODO.

```suggestion
            item["city"] = item["city"].title()
            item["street_address"] = item.pop("street")
````